app/(routes)/exam/page.tsx

```diff
@@ -149,65 +168,169 @@ export default function ExamHomePage() {
+            setCurrentUserData={setCurrentUserData}
+            nextViewState={incrementViewState}
+          />
+        )}
+        {viewState === VIEW_STATES.SCALES_NOTATE2 && (
+          <ScalesNotate2
+            currentUserData={currentUserData}
+            setCurrentUserData={setCurrentUserData}
+            nextViewState={incrementViewState}
+          />
+        )}
+        {viewState === VIEW_STATES.SCALES_NOTATE3 && (
+          <ScalesNotate3
+            currentUserData={currentUserData}
+            setCurrentUserData={setCurrentUserData}
+            nextViewState={incrementViewState}
+          />
+        )}
+        {viewState === VIEW_STATES.SCALES_NOTATE4 && (
+          <ScalesNotate4
+            currentUserData={currentUserData}
+            setCurrentUserData={setCurrentUserData}
+            nextViewState={incrementViewState}
+          />
+        )}
+        {viewState === VIEW_STATES.SCALES_NOTATE5 && (
+          <ScalesNotate5
+            currentUserData={currentUserData}
+            setCurrentUserData={setCurrentUserData}
+            nextViewState={incrementViewState}
+          />
+        )}
+        {viewState === VIEW_STATES.SCALES_NOTATE6 && (
+          <ScalesNotate6
+            currentUserData={currentUserData}
+            setCurrentUserData={setCurrentUserData}
+            nextViewState={incrementViewState}
+          />
+        )}
+        {viewState === VIEW_STATES.TRIADS_NOTATE && (
+          <TriadsNotate
+            currentUserData={currentUserData}
+            setCurrentUserData={setCurrentUserData}
+            nextViewState={incrementViewState}
+          />
+        )}
+        {viewState === VIEW_STATES.SEVENTH_CHORDS_NOTATE && (
+          <SeventhChordsNotate
+            currentUserData={currentUserData}
+            setCurrentUserData={setCurrentUserData}
+            nextViewState={incrementViewState}
+          />
+        )}
+        {viewState === VIEW_STATES.CHORDS_IDENTIFY && (
+          <ChordsIdentify
+            currentUserData={currentUserData}
+            setCurrentUserData={setCurrentUserData}
+            nextViewState={incrementViewState}
+          />
+        )}
+        {viewState === VIEW_STATES.WRITE_PROGRESSIONS && (
+          <WriteProgressions
+            currentUserData={currentUserData}
+            setCurrentUserData={setCurrentUserData}
+            nextViewState={incrementViewState}
+          />
+        )}
+        {viewState === VIEW_STATES.WRITE_BLUES_CHANGES && (
+          <WriteBluesChanges
+            currentUserData={currentUserData}
+            setCurrentUserData={setCurrentUserData}
+            nextViewState={incrementViewState}
+          />
+        )}
+        {viewState === VIEW_STATES.SUBMIT_AND_EXIT && (
+          <main className="flex min-h-[500px] flex-col items-center justify-center mt-12 gap-20">
+            <Typography variant="h3" marginLeft={8}>
+              Congratulations! You have completed the exam.
```

Comment:
Move the congratulations message to another page after the user has submitted the final test